compiler/rustc_error_messages/src/lib.rs

```diff
@@ -107,8 +107,8 @@ impl From<Vec<FluentError>> for TranslationBundleError {
 /// (overriding any conflicting messages).
 #[instrument(level = "trace")]
 pub fn fluent_bundle(
-    mut user_provided_sysroot: Option<PathBuf>,
-    mut sysroot_candidates: Vec<PathBuf>,
+    user_provided_sysroot: PathBuf,
```

Comment:
```suggestion
    sysroot: PathBuf,
```
It's not necessarily user-provided now.